### ISSUE TITLE
Cannot interact with express gateway admin while in the same path (PWD)

### ISSUE THREAD

**Issue by gianpaj** [ExpressGateway/express-gateway.io#217]
```
$ eg gateway create
? What's the name of your Express Gateway? expressg
? Where would you like to install your Express Gateway? expressg
? What type of Express Gateway do you want to create? Getting Started with Express Gateway
   create package.json
   create server.js
   create config/gateway.config.yml
   create config/system.config.yml
   create config/models/applications.json
   create config/models/credentials.json
   create config/models/users.json
...
+ express-gateway@1.7.3
added 775 packages in 13.818s

To start expressg, run the following commands:
    cd expressg && npm start

$ cd expressg
$ npm start
```

This works:
`curl http://localhost:8080/ip`

Then with a new terminal
```
$ cd expressg
$ eg users create
? Enter firstname [required]: first
? Enter lastname [required]: last
? Enter username [required]: firstlast
? Enter email: undefined
? Enter redirectUri: undefined
✖ <!DOCTYPE html>
<html lang="en">
<head>
<meta charset="utf-8">
<title>Error</title>
</head>
<body>
<pre>Cannot POST //users/</pre>
</body>
</html>
```

**Comment by gianpaj** [comment]
Sorry, should have opened in the correct project.
Anyway, this looks a like a duplicate of https://github.com/ExpressGateway/express-gateway/issues/669

**Comment by XVincentX** [comment]
@gianpaj This should now be solved in EG 1.7.4

Thanks for the issue though!

**Comment by XVincentX** [comment]
@gianpaj This should now be solved in EG 1.7.4

Thanks for the issue though!
